src/main.ts

```diff
@@ -1,8 +1,55 @@
-import { NestFactory } from '@nestjs/core';
-import { AppModule } from './app.module';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import { patchNestJsSwagger } from 'nestjs-zod';
+import { SpelunkerModule } from 'nestjs-spelunker';
+import * as fs from 'fs';
+import { dump } from 'js-yaml';
+
+import { createApp } from './app';
+
+import 'share/models/associations';
 
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
+  const app = await createApp();
```

Comment:
It would be better to move the logic for generating the dependency graph and the Swagger documentation into separate functions or modules. This would make the `bootstrap` function more readable and maintainable. [important]

Reply:
変えてみました！